packages/docz-core/src/commands/dev.ts

```diff
@@ -21,7 +22,26 @@ export const dev = async (args: Config) => {
   const server = await bundler.createServer(bundler.getConfig())
   const app = await server.start()
 
-  app.on('listening', async ({ server }) => {
+  app.on('listening', async ({ server, options }) => {
```

Comment:
I like to separate the concerns of each script, so I think that put bundler logic on `dev.ts` command is a mistake. Shouldn't be a function of a command update bundler's plugins.

And in this case, you can easily do something like we did to detect port of websockets:

```
const config = loadConfig(args)
const port = await detectPort(config.port)
const bundler = webpack({ ...config, port }, 'development')
```

Reply:
Make this change you can keep doing plugin configuration on bundler instead of in the `dev.ts` command!